.vscode/launch.json

```diff
@@ -15,7 +15,12 @@
             ],
             "outFiles": [
                 "${workspaceFolder}/out/**/*.js"
-            ]
+            ],
+            "preLaunchTask": "npm: compile",
+            "env": {
+                // Uncomment below line to show telemetry on Debug console
```

Comment:
should it be uncommented by default?

Reply:
this setting difficult to discover, and it would be great if all telemetry automatically shows up in debug console.

Reply:
When this Env variable is set then telemetry is not sent to the cluster and only shown on the console. I am afraid that the production telemetry may also stop working, if I uncomment it.